modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -4166,6 +4255,9 @@ private boolean _syncReleaseVersions(
 
 	private static final char _DEPENDENCY_KEY_SEPARATOR = '/';
 
+	private static final String _GENERATE_POM_INFO_TASK_NAME =
```

Comment:
Task names are public APIs (changing them is a breaking change), so the constant should be public. After making this change, run `gradlew baseline`.